ThingIFSDK/ThingIFSDK/Trait/TraitCommandForm.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  TraitCommandForm.swift
+//  ThingIFSDK
+//
+//  Created on 2016/12/08.
+//  Copyright (c) 2016 Kii. All rights reserved.
+//
+
+import Foundation
+
+/**
+Form of a command for trait version thing.
+
+This class contains data in order to create `Command` with
```

Comment:
@yongpingchen I think `TraitCommand` is not required.

Because server response of REST API to get command is same if we use non trait thing or trait thing.

What do you think?